emma/model/member.py

```diff
@@ -134,12 +134,12 @@ def extract(self):
 
         return extracted
 
-    def _add(self, signup_form_id):
+    def _add(self, signup_form_id, group_ids):
         """Add a single member"""
         path = '/members/add'
         data = self.extract()
-        if len(self.groups):
-            data['group_ids'] = self.groups.keys()
```

Comment:
I didn't see a way for someone to have set `self.groups` at this point (for a new member), other than by directly setting `_dict` on the MemberGroupCollection as is done in the tests and which seems like a bad idea. If we really want to make this backwardly compatible, I could change it to also include `self.groups.keys`.